src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -78,8 +78,10 @@ enum EventKey {
     utilities_viewMutationType,
     utilities_viewCopyNumber,
     utilities_discreteVsDiscreteTable,
-    utilities_stackedBarHorizontalBars,
-    utilities_showRegressionLine
+    utilities_horizontalBars,
+    utilities_showRegressionLine,
+    utilities_percentageBars,
+    utilities_stackedBars
```

Comment:
Code is yet to be cleaned up and tests needs to be updated/added. I'll work on it once we are good with layout.